db/schema.rb

```diff
@@ -254,12 +254,13 @@
   create_table "participations", force: :cascade do |t|
     t.string   "guid",                    limit: 255
     t.integer  "target_id",               limit: 4
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.